spec/spec_helper.rb

```diff
@@ -14,25 +29,30 @@
   config.filter_sensitive_data('API_TOKEN') { ENV['MASTER_JWT'] }
 end
 
-mode = ENV['MODE'] || 'unit'
```

Comment:
Removed the test mode now that we're using VCR

Reply:
VCR?